ArcadiaParties.CQRS/Queries/GetDetailedUserFromAssistantQuery.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ArcadiaParties.Data.Abstractions.DTOs;
+using MediatR;
+
+namespace ArcadiaParties.CQRS.Queries
+{
+    public class GetDetailedUserFromAssistantQuery :IRequest<DetailedUserFromAssistantDTO>
```

Comment:
You don't need to use the whole sentence in class names. You can omit prepositions and reorder words to get nicer name like: `GetAssistantEmployeeQuery`, `AssistantEmployeeDTO` and so on.

Reply:
Renamed

Reply:
Not everyone.